databases/week2/hw/part1.sql

```diff
@@ -0,0 +1,39 @@
+USE hyf_lesson2;
+select * from task;
+select * from user_task;
+#Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT  INTO task (title, description, created, updated, due_date, status_id)
```

Comment:
You were supposed to use the database from the first class ;P 

But it's ok, no need to change.